.github/workflows/ci.yml

```diff
@@ -144,7 +144,7 @@ jobs:
           - 24.0.9
           - 25.0.5
           - 26.1.4
-          - 27.2.1
+          - 27.3.0
```

Comment:
This currently installs v27.3.0-rc.1, because it's using the test-channel. It should automatically switch to v27.3.0 once that's released

Reply:
Yup; works; https://github.com/docker/compose/actions/runs/10855103277/job/30126964277?pr=12126

```
Run docker --version
Docker version 27.3.0-rc.1, build 48a2cdf
```